/jds/bin/diff -uprN flac-1.2.1.old/man/flac.1 flac-1.2.1/man/flac.1
--- flac-1.2.1.old/man/flac.1	2009-02-26 19:36:59.768978000 +0000
+++ flac-1.2.1/man/flac.1	2009-02-26 19:38:52.385476000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" This manpage has been automatically generated by docbook2man 
 .\" from a DocBook document.  This tool can be found at:
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
@@ -321,6 +322,20 @@ Force input (when encoding) or output (w
 .TP
 \fB--no-warnings-as-errors\fR
 These flags can be used to invert the sense of the corresponding normal option.
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWflac
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 .PP
 metaflac(1).
@@ -329,3 +344,4 @@ The programs are documented fully by HTM
 .SH "AUTHOR"
 .PP
 This manual page was written by Matt Zimmerman <mdz@debian.org> for the Debian GNU/Linux system (but may be used by others).
+...\" LSARC 2005/734 GNOME For Nevada
/jds/bin/diff -uprN flac-1.2.1.old/man/metaflac.1 flac-1.2.1/man/metaflac.1
--- flac-1.2.1.old/man/metaflac.1	2009-02-26 19:36:59.804143000 +0000
+++ flac-1.2.1/man/metaflac.1	2009-02-26 19:39:27.377379000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" This manpage has been automatically generated by docbook2man 
 .\" from a DocBook document.  This tool can be found at:
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
@@ -294,6 +295,21 @@ Merge adjacent PADDING blocks into singl
 \fB--sort-padding\fR
 Move all PADDING blocks to the end of the metadata and merge them
 into a single block.
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWflac
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 .PP
 flac(1).
+...\" LSARC 2005/734 GNOME For Nevada
